models/user/email_address.go

```diff
@@ -486,7 +486,7 @@ func ActivateUserEmail(ctx context.Context, userID int64, email string, activate
 
 	// Activate/deactivate a user's primary email address and account
 	if addr.IsPrimary {
-		user, exist, err := db.Get[User](ctx, builder.Eq{"id": userID, "email": email})
+		user, exist, err := db.Get[User](ctx, builder.Eq{"id": userID, "email": strings.ToLower(email)})
```

Comment:
You should do :

```
user := Get(id=userID)
if lower(user.email) != lower(email) { not exist }
```

Reply:
Done in 4a03afc086db511fb4f230b250824d3414bc9306